### PR TITLE
[PORT] plane_master atomized + 515.1543 and higher BYOND compatibility

### PR DESCRIPTION
**Pull request by ariaworld**
## About The Pull Request
PORT: https://github.com/BeeStation/BeeStation-Hornet/pull/4567 (Only the most important things, notably the plane master controller, eye blur update, rotatium reagent fix)
Also ported BeeStation's Destroy() method.
And 515.1543+ compatibility

## Why It's Good For The Game
Citmain's filters/plane code is quite messy, it seems like many things were supposed to be ported from TG but then manual modifications were made, which makes porting stuff from TG more difficult, this hopefully is the first step in the right direction.

## Changelog
:cl:
code: Ports TG's plane_master_controller.
/:cl:

### CODE REVIEW

File: code/modules/admin/view_variables/debug_variable_appearance.dm
Comment (SandPoot) [inline code/modules/admin/view_variables/debug_variable_appearance.dm:1]: This whole file is regression

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -160,9 +160,6 @@ GLOBAL_VAR_INIT(embedpocalypse, FALSE) // if true, all items will be able to emb
 	var/list/grind_results //A reagent list containing the reagents this item produces when ground up in a grinder - this can be an empty list to allow for reagent transferring only
 	var/list/juice_results //A reagent list containing blah blah... but when JUICED in a grinder!
 
-	//the outline filter on hover
-	var/outline_filter
```
Comment (SandPoot) [inline code/game/objects/items.dm:164]: What happened here? just want to know

Comment (ariaworld) [reply code/game/objects/items.dm:164]: Unused var, removed just like on port PR

File: code/modules/admin/view_variables/debug_variable_appearance.dm
Comment (SandPoot) [reply code/modules/admin/view_variables/debug_variable_appearance.dm:1]: Solved myself.

**Review by SandPoot — APPROVED** [review]

**Pull request merged**